rbac/rbac/settings.py

```diff
@@ -397,28 +397,33 @@
 if KAFKA_ENABLED:
     KAFKA_AUTH = {}
     if ENVIRONMENT.bool("CLOWDER_ENABLED", default=False):
-        kafka_broker = LoadedConfig.kafka.brokers[0]
-        KAFKA_HOST = kafka_broker.hostname
-        KAFKA_PORT = kafka_broker.port
-        try:
-            if kafka_broker.authtype.value == "sasl":
-                KAFKA_AUTH.update(
-                    {
-                        "bootstrap_servers": f"{KAFKA_HOST}:{KAFKA_PORT}",
-                        "sasl_plain_username": kafka_broker.sasl.username,
-                        "sasl_plain_password": kafka_broker.sasl.password,
-                        "sasl_mechanism": kafka_broker.sasl.saslMechanism.upper(),
-                        "security_protocol": kafka_broker.sasl.securityProtocol.upper(),
-                    }
-                )
-            if kafka_broker.cacert:
-                KAFKA_AUTH["ssl_cafile"] = LoadedConfig.kafka_ca()
-        except AttributeError:
-            KAFKA_AUTH = {}
+        kafka_broker = LoadedConfig.kafka.brokers
+        if len(kafka_broker) == 0:
+            ValueError("No kafka brokers available")
+        for i in range(0, len(kafka_broker)):
+            kafka_host = kafka_broker[i].hostname
+            kafka_port = kafka_broker[i].port
+
+            try:
+                if kafka_broker[i].authtype.value == "sasl":
+                    KAFKA_AUTH.update(
+                        {
+                            "bootstrap_servers": f"{kafka_host}:{kafka_port}",
+                            "sasl_plain_username": kafka_broker[i].sasl.username,
+                            "sasl_plain_password": kafka_broker[i].sasl.password,
+                            "sasl_mechanism": kafka_broker[i].sasl.saslMechanism.upper(),
+                            "security_protocol": kafka_broker[i].sasl.securityProtocol.upper(),
+                        }
+                    )
+                if kafka_broker[i].cacert:
+                    KAFKA_AUTH["ssl_cafile"] = LoadedConfig.kafka_ca()
+            except AttributeError:
+                KAFKA_AUTH = {}
     else:
-        KAFKA_HOST = "localhost"
-        KAFKA_PORT = "9092"
-    KAFKA_SERVER = f"{KAFKA_HOST}:{KAFKA_PORT}"
+        kafka_host = "localhost"
+        kafka_port = "9092"
+
+    KAFKA_SERVER = f"{kafka_host}:{kafka_port}"
```

Comment:
@Ellen-Yi-Dong does this contains array ?

I did not run code but It looks we need to build array here from `LoadedConfig.kafka.brokers` but this looks to me that it takes just last element from that array. Same for KAFKA_AUTH. 

This var  KAFKA_SERVER is passed [here](https://github.com/RedHatInsights/insights-rbac/blob/9095381e70e2e0ac86c728e2985b654594499288/rbac/core/kafka.py#L44)  to kafka client and maybe it should be called accordingly:  `KAFKA_SERVERS`.

Also If it was not yet -  I recommend to test this change in eph. env.  - let me know if you need some guidance.

Thanks!

Reply:
FROM [docs](https://kafka-python.readthedocs.io/en/master/apidoc/KafkaProducer.html):  

> bootstrap_servers – ‘host[:port]’ string (or list of ‘host[:port]’ strings) that the producer should contact to bootstrap initial cluster metadata. This does not have to be the full node list. It just needs to have at least one broker that will respond to a Metadata API Request. Default port is 9092. If no servers are specified, will default to localhost:9092.

And I believe that list(array) what we exactly need for this change as I was reading related email.

Let me know what do you  think about this, thanks!

Reply:
Hello Libor! Thank you for the review and information! ! 
I think that it makes sense, and I'll get those changes in! (Though might reach out again for questions)